net-analyzer/nuttcp/files/nuttcp.initd

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+command="/usr/bin/nuttcp"
+command_background="yes"
+pidfile="/run/${SVCNAME}.pid"
+command_args="${NUTTCP_OPTS}"
+
+depend() {
+	need net
+	provide nuttcp
+}
+
+start_post() {
+	pgrep -n $(echo ${SVCNAME} | sed 's/\..*//g') > ${pidfile}
```

Comment:
Why do you need to do that? I think ssd has an option to write pidfile for you, doesn't it?

Reply:
I need a possibility to launch multiple instances of nuttcp.

I could replace "$(echo ${SVCNAME} | sed 's/\..*//g')" with ${SVCNAME%%.*}, but it will be a bashism.